### DISCUSSION TITLE
v23 alpha2 发现僵尸进程

### DISCUSSION THREAD

**Discussion by Linux-qitong** [linuxdeepin/developer-center#4189] [Troubleshooting | 故障排查]
https://bbs.deepin.org/phone/zh/post/255617
还有 登陆系统后 会有 桌面黑屏情况

系统日志报错：
![202304232306435250_image](https://user-images.githubusercontent.com/130018882/233872882-733e01d6-5fd6-4553-9b15-955fd148d2d3.png)
![202304232304206199_image](https://user-images.githubusercontent.com/130018882/233872886-bf4bbac7-63ef-40d9-8203-22876b8dc13f.png)

**Comment by BLumia** [comment]
已知问题，部分已修复（可能还没推送到内测仓库）
